common/src/main/java/feast/common/logging/AuditLogger.java

```diff
@@ -112,24 +129,47 @@ private static void log(Level level, AuditLogEntry entry) {
       return;
     }
 
-    // Log event to audit log through enabled formats
-    String entryJSON = entry.toJSON();
-    switch (level) {
-      case TRACE:
-        log.trace(AUDIT_MARKER, entryJSON);
-        break;
-      case DEBUG:
-        log.debug(AUDIT_MARKER, entryJSON);
-        break;
-      case INFO:
-        log.info(AUDIT_MARKER, entryJSON);
-        break;
-      case WARN:
-        log.warn(AUDIT_MARKER, entryJSON);
-        break;
-      case ERROR:
-        log.error(AUDIT_MARKER, entryJSON);
-        break;
+    // Either forward log to logging layer or log to console
+    if (properties.getMessageLogging().getDestination().equals(FLUENTD_DESTINATION)) {
```

Comment:
What prevents this code from executing if message Logging is disabled?

Reply:
That's handled [here](https://github.com/feast-dev/feast/blob/master/common/src/main/java/feast/common/logging/interceptors/GrpcMessageInterceptor.java#L73) from #940.

Reply:
By the way, as a convention we try and instantiate variables from getters in order to make code more readable. So something like
```
String messageLogDestination = properties.getMessageLogging().getDestination();
if (message.equals(FLUENTD_DESTINATION)) {

}
```